awscrt/http.py

```diff
@@ -71,12 +74,19 @@ def new(cls, bootstrap, host_name, port, socket_options,
         """
         future = Future()
         try:
-            assert isinstance(bootstrap, ClientBootstrap)
-            assert host_name
-            assert tls_connection_options is None or isinstance(tls_connection_options, TlsConnectionOptions)
+            assert isinstance(bootstrap, ClientBootstrap) or bootstrap is None
+            assert isinstance_str(host_name) and host_name
```

Comment:
comment this nephew

Reply:
Justin also thought this looked weird. Will comment